lib/pxconfig.js

```diff
@@ -21,6 +22,10 @@ class PxConfig {
         if (this.config.PROXY_URL) {
             this.config.agent = new HttpsProxyAgent(this.config.PROXY_URL);
         }
+        else if (this.config.HTTPS_AGENT_KEEP_ALIVE_TIMEOUT_MS !== 0) {
+            this.config.agent = new https.Agent({ keepAlive: true, timeout: this.config.HTTPS_KEEP_ALIVE_AGENT_TIMEOUT_MS });
```

Comment:
Didn't you mention another field?

Reply:
I think this is the correct field, but take a look here again and tell me if I'm wrong?

https://nodejs.org/api/http.html#class-httpagent

Reply:
ok